generators/init/templates/package.json.ejs

```diff
@@ -32,7 +32,7 @@
     "prettier-plugin-packagejson": "<%= this.nodeDependencies['prettier-plugin-packagejson'] %>"
   },
   "engines": {
-    "node": ">=<%= NODE_VERSION %>"
+    "node": ">=<%= nodeVersion %>"
```

Comment:
Why isn't it in constant format (for my own culture)?

Reply:
Used to be, but with typescript, I don't fell we should add a field called NODE_VERSION to the type..

Reply:
Most of constants will be changed to variables.
Versions will be loaded from pom and Dockerfile.